test/unit_tests/controllers/account_controller_test.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get/get.dart';
+import 'package:mockito/mockito.dart';
+import 'package:pispapp/controllers/app/account_controller.dart';
+import 'package:pispapp/controllers/app/auth_controller.dart';
+import 'package:pispapp/models/user.dart';
+import 'package:pispapp/repositories/auth_repository.dart';
+import 'package:pispapp/repositories/interfaces/i_account_repository.dart';
+
+class MockAccountRepository extends Mock implements IAccountRepository {}
+
+class MockAuthRepository extends Mock implements AuthRepository {}
```

Comment:
why this part implements `AuthRepository` but the previous one implements `IAccountRepository` ?

Reply:
No interface for AuthRepository.